packages/data-explorer-ui/src/components/common/Banner/banner.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { Typography } from "@mui/material";
+import { Meta, StoryFn } from "@storybook/react";
+import React from "react";
+import { CallToActionButtonWithCustomComponentStory } from "../Button/components/CallToActionButton/callToActionButton.stories";
+import { Banner } from "./banner.styles";
+
+export default {
+  argTypes: {
+    children: { control: { disabled: true } },
+  },
+  component: Banner,
+  parameters: {
+    layout: "fullscreen",
+  },
+  title: "Components/Common/Banner",
+} as Meta<typeof Banner>;
+
+const BannerTemplate: StoryFn<typeof Banner> = (args) => <Banner {...args} />;
+
+export const BannerStory = BannerTemplate.bind({});
+BannerStory.args = {
+  children: (
+    <>
```

Comment:
Change the `children` argument to:

```
    <Typography component="span" variant="text-body-small-400">
      Optional announcements text goes here followed by a{" "}
      <CallToActionButtonStory
        ButtonElType={TextButtonWhiteStory}
        callToAction={{
          label: "Call to Action",
          url: "https://www.google.com",
        }}
      />
    </Typography>
```

I will make a separate ticket to make a new button component that inherits the parent typography styles.